### DISCUSSION TITLE
Share/Download the chat

### DISCUSSION THREAD

**Discussion by ivanhercaz** [Taewan-P/gpt_mobile#127] [Ideas]
Hi!
First of all, thank you very much for this app and your work.

Could it be possible to add a way to share or download the conversation?

Just a few ideas:

## UI

- It may be an option of the three-dots button on the right of the AppBar of a chat to save space.
- It may be a new button on the chats screens when some chat is selected, to the left of the thrash icon.

## Functionality

- Share will be a complete feature in which the smartphone share dialog appears.
- The chat may be a PDF, but maybe an easier option and more comfortable to share it is to convert it to markdown.
- If PDF will be required, I think it may be easier to create it based from the markdown.
- What worries me about include a PDF option is that maybe it requires to include some big dependency only for that. I really don't know Kotlin a lot, for smartphones I usually develop on Dart/Flutter.

Thanks in advance just for taking your time reading my request!

**Comment by nanderer** [comment]
just plain html or markdown would be fine i guess, i hate pdf files especially on the phone ;) if you need pdf you then can use some print app like this one https://f-droid.org/de/packages/org.billthefarmer.print/ but dont waste time to generate pages in a world where we are beyond such a thing. Printing stuff kills our planets lung...

**Comment by Taewan-P** [comment]
Hi. Export chat functionality is already implemented (with markdown)
